### ISSUE TITLE
MarkdownStyleSheet p textColor is white

### ISSUE THREAD

**Issue by nilsreichardt** [flutter/flutter_markdown#198]
If you create a new MarkdownStyleSheet, change p and not set a font color, the font color is always white.

```
MarkdownBody(
   data: text,
   styleSheet: MarkdownStyleSheet(p: flowingText),
);
```

The font color should be picked by the theme. So with this you have the right font color, no matter with brightness.

**Comment by bierbaumtim** [comment]
With #210  I fixed some issues on MarkdownStyleSheet and can't reproduce the issue. Also there are no hardcoded default colors. So could you check if the issue still exists after #210  is merged and published. 
If it still exists it would be helpfull if you can provide a small example which reproduces this issue.

**Comment by AndroidNils** [comment]
@bierbaumtim No, the bug isn't fixed. Try it out:

```
import 'package:flutter/material.dart';
import 'package:flutter_markdown/flutter_markdown.dart';

void main() {
  runApp(MyApp());
}

class MyApp extends StatelessWidget {
  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: 'Demo',
      home: Scaffold(
        body: SafeArea(
          child: MarkdownBody(
            data: 'Test',
            styleSheet: MarkdownStyleSheet(p: TextStyle(fontSize: 16)),
          ),
        ),
      ),
    );
  }
}
```

If you set `Scaffold.backgroundColor` to `Colors.black`, you will see that `Test` is white.

**Comment by mjordan56** [comment]
@domesticmouse This issue was fixed by PR #243.
